### ISSUE TITLE
Update to v1.4.4

### ISSUE THREAD

**Issue by Tissy** [jumbo77/portapackh2#4]
Loving your work as this is the only solution I found to get my PortaPack H2+ working.

Please could you update to v1.4.4 when you get the chance :-)

https://github.com/eried/portapack-mayhem/releases/tag/v1.4.4

**Comment by viorellu** [comment]
After attemtping to update to 1.4.4 my chinese hackRF clone, the portapack H2 no longer worked.
Your version saved the day! Many thanks!

**Comment by QioTek** [comment]
This is the bad news. I have the same clone board.

**Comment by matthewric** [comment]
Latest 1.5.1

[portapack-h1_h2-mayhem_1_5_1.zip](https://github.com/jumbo77/portapackh2/files/8568775/portapack-h1_h2-
mayhem_1_5_1.zip)

**Comment by jumbo5566** [comment]
OK i add this files

**Comment by onetechct** [comment]
> After attempting to update to 1.4.4 my chinese hackRF clone, the portapack H2 no longer worked. The HackRF itselfs supports 1.4.4 as a standalone unit, it's only the portapack that fails to start with 1.4.4. Your version saved the day! Many thanks!
> 
> So I guess I should add "This version is a valid solution for unbricking hackRFwith portapack H2 aquired from Bangood"

I have same problem! Can't use any version of https://github.com/eried/portapack-mayhem/releases/
Last working is 1.4.3 but only from jumbo, the other repository not work. I must use old 1.4.0 for dfu recovery

**Comment by jumbo77** [comment]
if don't work hold and Press LEFT key  then power on,i tested v1.5.4 mayhem ok.

**Comment by hebeme** [comment]
how can i flash the latest mayhem master branch? I have tried, however, The screen is just black.
to be clear I want to be able to go to:
https://github.com/eried/portapack-mayhem/releases/tag/v1.5.4
and install from there.

How can this be accomplished?

Thanks for your time.

**Comment by tobishi69** [comment]
Hi , i got a unit hackrfone+h2 portapack, and seller give me your link for the firmware 1.4.2 actually is loaded.But the seller
say   i can not put 1.5.4  mayhem , and i must use 1.4.2 .So i ask  some details about this upgrades,how i can not (for seller) the 1.5.4 , some people got a black screen Why ? Whatare the difference from 1.4.2 and 1.5.4 ?  new features /  Bug fixed ? I  look to have on receive menu DMR decoding , much seller have on ebay but only in photo,you  can help me? Please @jumbo77
Wait your  help/comment ,thank you,;)

**Comment by onetechct** [comment]
Read well wiki for change boot display driver

Inviato da Outlook per iOS<https://aka.ms/o0ukef>
________________________________
Da: tobishi69 ***@***.***>
Inviato: Tuesday, November 8, 2022 10:39:39 AM
A: jumbo77/portapackh2 ***@***.***>
Cc: onetechct ***@***.***>; Comment ***@***.***>
Oggetto: Re: [jumbo77/portapackh2] Update to v1.4.4 (Issue #4)


Hi , i got a unit hackrfone+h2 portapack, and seller give me your link for the firmware 1.4.2 actually is loaded.But the seller
say i can not put 1.5.4 mayhem , and i must use 1.4.2 .So i ask some details about this upgrades,how i can not (for seller) the 1.5.4 , some people got a black screen Why ? Whatare the difference from 1.4.2 and 1.5.4 ? new features / Bug fixed ? I look to have on receive menu DMR decoding , much seller have on ebay but only in photo,you can help me? Please @jumbo77<https://github.com/jumbo77>
Wait your help/comment ,thank you,;)

**Comment by tobishi69** [comment]
@onetechct have  readed wiki , but have readed also someonecan't solve after upgrade 1.4.2 to 1.5.4 because stuck on  black screen so i ask some    indication about improvements / fix of   1.5.4 respect 1.4.2 and if   really is possible have a system    bricked,thank you at all want/desire give   information about firmware changes

**Comment by jumbo77** [comment]
pls check your protapack h2 is the same with me v3.6 pcb
https://www.mmdvm.club/index.php/h2plus.html

**Comment by tobishi69** [comment]
Hi @jumbo77 here attached my board photo, confirm is exactly your v 3.6
so can you explain me how can install and / or modify mayhem 1.5.4 and future version in this board ?
![tobishi_IMG_20221109_203714](https://user-images.githubusercontent.com/117631476/201027655-1045d285-f233-4a8b-984f-4ee5d991b7ed.jpg)

**Comment by jumbo77** [comment]
yes,i need test the mayhem 1.5.4 first,pls wait.

**Comment by tobishi69** [comment]
Thank you , you know difference from 1.4.2 actually  i've  and 1.5.4 ? I'd like learn how compile source and create my own firmware :)

**Comment by jumbo77** [comment]
[https://www.mmdvm.club/usr/uploads/2022/11/612509093.jpg](https://www.mmdvm.club/usr/uploads/2022/11/612509093.jpg)  i already tested it is ok,
//change 
//portapack-mayhem/blob/next/firmware/application/portapack.cpp
//delete follow ---------------------
if (switches_state[(size_t)ui::KeyEvent::Up]){
			save_config(1);
			// model = PortaPackModel::R2_20170522; // Commented these out as they should be set down below anyway
		}
		else if (switches_state[(size_t)ui::KeyEvent::Down]){
			save_config(2);
			// model = PortaPackModel::R1_20150901;
		}
		else if (switches_state[(size_t)ui::KeyEvent::Left]){
			save_config(3);
			// model = PortaPackModel::R1_20150901;
		}
		else if (switches_state[(size_t)ui::KeyEvent::Right]){
			save_config(4);
			// model = PortaPackModel::R2_20170522;
		}
		else if (switches_state[(size_t)ui::KeyEvent::Select]){
			save_config(0);
		}
//----------------------------------------------------
//then 
//use 
save_config(3); 
//instead

**Comment by tobishi69** [comment]
Yeah !! , i 'm new for customize so , can give me some indication about tools to install ,andhow use source to compile ?

**Comment by tobishi69** [comment]
you have tested in  both mode portapack / Hackrf one and tried all apps ,  working ? no  crash ,freeze, more speed ?        new features i'm interested to receiver/scanner feature of portpack can post shots? thanks , great job, you  can  post a modded 1.5.4  ready  to flash included sd files package

**Comment by tobishi69** [comment]
@jumbo77 Hallo , have news for me ? also  all indication to update  my own so i learn something about  firmware updates and microsd improvement ,  i need  best source where get files to put on my sd with actual firmware 1.4.0 or better                      giveme a   read to flash 1.5.4 sd files included.Thank you all frined want give help me,Cheers.Simone

**Comment by tobishi69** [comment]
@jumbo77 and friends , Hi , here Tobishi , i've a question about new 1.6.0 firmware  : it support us 3.6v mmdvm type board ? 
Or is necessary make some scripts or changes at actual 1.4.2 ? Please write me indication if can update actual firmware stuck on 1.4.2 because have fear to get a brick. Hope someone friend or entusiast of Hackrf Portapack h2+ give some tips and trick useful at comunity.I got 2ns unit from diyradio on ebay , but support is poor and give links without a concrete testing.Very great at all contributes of this community.Bye , Simone (Italy)
